live-examples/html-examples/text-content/css/dl.css

```diff
@@ -0,0 +1,14 @@
+@font-face {
+    font-family: "Fira Sans";
+    src: local("FiraSans-Regular"),
+    url("/media/fonts/FiraSans-Regular.woff2") format("woff2");
+}
+
+p, dl {
+    font: 1rem "Fira Sans", sans-serif;
+}
+
+dl > dt {
+    font-weight: bold;
+    text-decoration: underline;
+}
```

Comment:
Nit: Missing empty line at end of file

Reply:
Weird, my editor is configured to alert me if I do that. Fixing it.